git-day-1/README.md

```diff
@@ -100,18 +98,7 @@ A volunteer will need to navigate and let the group guide them in following thro
 - Trainees do this section in pairs
 - ⏲️ 10 mins
 
-Next, create a branch so that your local git repository looks like this:
-
-```mermaid
----
-title: cyf-demo-repo
----
-gitGraph
-   commit
-   commit
-   commit
```

Comment:
I agree about this mermaid diagram - removing it for now but will need to figure out a better way of showing `week-1-workshop` pointing to `main`.